sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -725,6 +725,13 @@ def process(self, element):
     partitions.append(latest_partition.files)
 
     if len(partitions) > 1:
+      # Copy jobs with BigQuery API do not support having
+      # a table partition as the destination
+      if '$' in destination:
```

Comment:
Is there a better way to identify that the table is partitioned beyond looking for a $?

Reply:
From what I have found online, you can tell if a given table is partitioned by performing a query: https://stackoverflow.com/questions/44881100/is-there-a-metadata-table-to-check-if-the-table-in-bigquery-is-partitioned

However, I did not find another way to tell if the destination itself is a partition of a table. BigQuery recognizes the `$` as a partition decorator: https://cloud.google.com/bigquery/docs/managing-partitioned-table-data#partition_decorators

Reply:
Hmm. I would reach out to the BQ team on this. looking for $ feels like it could be inconsistent.